specificatie/openapi.yaml

```diff
@@ -85,6 +85,7 @@ paths:
 
         Gebruik de expand-parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources nummeraanduiding, woonplaats, openbare ruimte en adresseerbaarobject, 
         zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/expand.feature).
+        Het is niet mogelijk om adresseerbare objecten te expanden wanneer de query parameter adresseerbaarObjectIdentificatie wordt gebruikt.
```

Comment:
Waarom is dat in dat geval niet mogelijk?

Reply:
In theorie is het mogelijk, maar we zullen dit aan de provider kant niet ondersteunen. De reden hiervoor is, dat als je adressen opvraagt met een adressseerbaar object ID, dan kunnen er meerdere adressen worden teruggegeven (hoofd- en nevenadressen). Als er dan wordt geëxpand op adresseerbaarobject dan bevat ieder adres onder _embedded de adresseerbare object informatie. Omdat dat allemaal dezelfde adresseerbare object informatie is, krijg je veel redundante informatie en dat willen we voorkomen.

Reply:
Ik heb hiervoor een scenario toegevoegd aan de feature file.